app/controllers/pdfs.rb

```diff
@@ -20,6 +21,20 @@ class App < Roda
             routing.redirect '/auth/login'
           end
         end
+
+        # POST /pdfs/
+        routing.post do
+          filename = routing.params['pdf_file'][:filename]
+          file = routing.params['pdf_file'][:tempfile]
+          content = Base64.strict_encode64(file.read)
```

Comment:
Would it be better to encode in service rather than in controller?

Reply:
fixed